.github/workflows/k8s-test-run.yml

```diff
@@ -0,0 +1,26 @@
+name: Weekly k8s test run
+on:
+  schedule:
+    # At 09:00:00 AM, every week on Monday, every month
+    - cron: '0 09 * * MON'
```

Comment:
Just as a recommendation, this is in UTC and would mean 12 PM for Bucharest time. Nothing wrong with this, but I would consider perhaps moving this to a less active hours, such as 5-6 AM Bucharest time (2-3AM UTC)

Reply:
@avirtopeanu-ionos thank you, I agree, I modified the time on this PR. Note that, for all other periodic workflows, we have the same hour: `09:00 AM`. I will note and maybe in the future we will also modify the hour for the rest of the periodic workflows.

Reply:
@avirtopeanu-ionos @mimihalescu done in https://github.com/ionos-cloud/terraform-provider-ionoscloud/pull/329/commits/05a9a6921bc39169c2e17333f7b0e79d21710be2